src/App.js

```diff
@@ -36,7 +36,8 @@ const client = new AWSAppSyncClient({
   auth: {
     type: appSyncConfig.authenticationType,
     apiKey: appSyncConfig.apiKey,
-  }
+  },
+  disableOffline: true
```

Comment:
See here: https://stackoverflow.com/questions/49118378/apollo-mutate-calling-update-four-times-for-a-single-mutation